From 85f535f9f7f1d3229513fad6443dfa0411315e93 Mon Sep 17 00:00:00 2001
From: Masayuki Igawa <masayuki@igawa.io>
Date: Thu, 21 Feb 2019 15:21:43 +0900
Subject: [PATCH] Fix typo in .stestr.conf

This commit fixes a typo in .stestr.conf.

Also fixes get_opt_lists which returns invalid
opts.

Change-Id: I2f453718ff5aeb688ecae48b7b9e409b614da7c2
---
 .stestr.conf                     | 2 +-
 vitrage_tempest_plugin/plugin.py | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/.stestr.conf b/.stestr.conf
index 6dcb542..fc7943e 100644
--- a/.stestr.conf
+++ b/.stestr.conf
@@ -1,3 +1,3 @@
 [DEFAULT]
-test_path=./virtage_tempest_plugin/tests
+test_path=./vitrage_tempest_plugin/tests
 top_dir=./
diff --git a/vitrage_tempest_plugin/plugin.py b/vitrage_tempest_plugin/plugin.py
index 437d314..18f26e2 100644
--- a/vitrage_tempest_plugin/plugin.py
+++ b/vitrage_tempest_plugin/plugin.py
@@ -43,4 +43,4 @@ class VitrageTempestPlugin(plugins.TempestPlugin):
         return [(config_rca_service.rca_service_group.name,
                  config_rca_service.RcaServiceGroup),
                 (config_rca_service.service_available_group.name,
-                 [config_rca_service.ServiceAvailableGroup])]
+                 config_rca_service.ServiceAvailableGroup)]
-- 
2.17.1

